docs/building/windows-instructions.md

```diff
@@ -5,13 +5,59 @@ You can build ML.NET either via the command line or by using Visual Studio.
 
 ## Required Software
 
-1. **[Visual Studio 2019 Version 16.4+](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise)** The Community version is completely free. The below build instructions were verified for VS 16.4.
-2. **[CMake](https://cmake.org/)** must be installed from [the CMake download page](https://cmake.org/download/#latest) and added to your path. If you want to use Visual Studio 2022, you need to be using at least CMake 3.21.
+1. **[Visual Studio 2019 Version 16.4+](https://www.visualstudio.com/downloads/) or [Visual Studio 2022](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise)** The Community version is completely free. The below build instructions were verified for VS 16.4 and higher.
+2. **[CMake](https://cmake.org/)** must be installed from [the CMake download page](https://cmake.org/download/#latest) and added to your path. 
+   * For Visual Studio 2019: CMake 3.14 or higher is required.
+   * For Visual Studio 2022: CMake 3.21 or higher is required.
+   * CMake 4.0 and later versions are supported.
 
-### Visual Studio 2019 Installation
+### Visual Studio Installation
```

Comment:
Can we generalize these steps to not be version specific, but instead mention similar steps are required for all versions of VS (2019, 2022, 2026)